lib/tf_lock/release.py

```diff
@@ -184,8 +187,13 @@ def main() -> None:
     from os import environ
 
     with sh.quiet():
+        successes = 0
         for path in paths:
-            tf_lock_release(path, env=environ.copy())
+            if tf_lock_release(path, env=environ.copy()) is None:
```

Comment:
You're dropping the UserError on the ground if/when it's returned.

Reply:
It gets chalked up as a failure, I think the errors are written out and that's handled seperately, we can just have it return 1 or 0 and just add that?